modules/microshift-mDNS.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: CONCEPT
+[id="microshift-mDNS_{context}"]
+== mDNS
+mDNS is a protocol used to allow name resolution and service discovery within a Local Area Network (LAN) using multicast exposed on the `5353/UDP` port.
+
+{product-title} includes an embedded mDNS server for deployment scenarios in which the authoritative DNS server cannot be reconfigured to point clients to services on {product-title}.
+
+This allows `.local` domains exposed by {product-title} to be discovered by other elements on the LAN.
```

Comment:
"This allows `.local`..." seems a little orphaned conceptually :/ can you incorporate it more with the preceding paragraph so the "this" isn't questionable?